src/main/webapp/js/user-page-loader.js

```diff
@@ -70,60 +70,26 @@ function showMessageFormIfViewingSelf() {
 
 /** Fetches messages and add them to the page. */
 function fetchMessages() {
-  // const url = '/messages?user=' + parameterUsername;
-  // fetch(url)
-  //     .then((response) => {
-  //       return response.json();
-  //     })
-  //     .then((messages) => {
-  //       const messagesContainer = document.getElementById('message-container');
-  //       if (messages.length == 0) {
-  //         messagesContainer.innerHTML = '<p>This user has no posts yet.</p>';
-  //       } else {
-  //         messagesContainer.innerHTML = '';
-  //       }
-  //       messages.forEach((message) => {
-  //         const messageDiv = buildMessageDiv(message);
-  //         messagesContainer.appendChild(messageDiv);
-  //       });
-  //     });
-  //const div = document.getElementById('lostDiv');
         const div = document.getElementById('message-container');
         while(div.firstChild){
             div.removeChild(div.firstChild);
         }
-        // var addDiv = document.createElement('div');
-        // addDiv.setAttribute('id', `message-container-0`);
-        // document.getElementById('lostDiv').appendChild(addDiv);
          messageContainer = document.getElementById(`message-container`);
-        // messageContainer.classList.add('message-container');
 
         const url = '/messages?user=' + parameterUsername;
         fetch(url).then((response) => {
           return response.json();
         }).then((messages) => {
-          // var counter = 0;
-          // var divCount = 0;
           if(messages.length == 0){
            messageContainer.innerHTML = '<p>There are no posts yet.</p>';
           }
           else{
            messageContainer.innerHTML = '';  
           }
-          //var counter = 0;
           messages.forEach((message) => {  
               const messageDiv = buildSummaryDiv(message);
```

Comment:
Does the 'buildSummaryDiv(..)' function still exist? I thought it got deleted on line 84 of the first file (index.html).

Reply:
Haha whoops nvm I was just looking at one commit.